sandbox/handlers/batch_send.js

```diff
@@ -85,7 +85,7 @@ export async function batch_send(req, res, next) {
 
   // Note: the docker container uses node:12 which does not support optional chaining
   const odsCodes = messages.map((message) => {
-    if (message && message.originator) {
+    if (message?.originator) {
```

Comment:
I think this would suffice
```suggestion
   return message?.originator?.odsCode
```

Reply:
```
const odsCodes = messages.map((message) => ( message?.originator?.odsCode ))
```